### DISCUSSION TITLE
PROBLEM  WITH IMPORTING DATA FROM QFIELD (SMARTPHONE) TO QGIS (COMPUTER)

### DISCUSSION THREAD

**Discussion by nunoymartins** [opengisch/QField#5496] [Bugs]
I copy the folder on the smartphone to a folder on the computer, then synchronize it with the plugin but it doesn't update. Data collected in the field is not transferred. I use QGIS, version 3.16.5-Hannover.

**Comment by fantan61** [comment]
Hello, maybe you have exceeded your data allowance. Check out https://qgiscloud.com
either buy storage or 
ou could try to get your data from the mobile device by going to Settings > Open Project Folder > data.gpkg, then use the three dots on the right side and the function "Send to..." in order to send your data to your computer, and paste the data into the original layers in the project, and re-synchronize the project with QFieldCloud.
good luck

Symbol „Von der Community überprüft“

**Comment by AveryAndrews** [comment]
Did you copy the folder from Android/data ... <your project folder>, or did you export first?  In my (recent) experience, export to another folder on the phone can be surprisingly slow, and doesn't finish if interrupted in various ways, and is likely to leave out the crucial .qgs file.
